TRANSFORMATIONS.md

```diff
@@ -0,0 +1,32 @@
+# Transformations
+
+When a new `Processor` is requested using the endpoint `/api/v1/bridges/{id}/processors` you can optionally provide a `Transformation`
+to modify the `Event` sent to the [Action](ACTIONS.md) defined on the `Processor`.
+
+You can use a `Transformation` to construct an entirely new `Event`, or selectively pass through parts of the original `Event`
+to your [Action](ACTIONS.md)
+
+If you do not specify a `Transformation` then the `Event` is passed through unchanged to your [Action](ACTIONS.md).
+
+## Writing a Transformation
+
+`Transformations` are defined using the [Qute Templating](https://quarkus.pro/guides/qute-reference) engine. You can provide a
+template to be used to transform the original `Event` as part of your `Processor` definition:
```

Comment:
Can we add an example of transforming the JSON to a plain string? To me it wasn't obvious it was possible

Reply:
@lucamolteni Just reading your comment, is the example below clear enough?

Reply:
Yes